src/api/DataSource/types.ts

```diff
@@ -86,6 +87,7 @@ export interface DataSourceUserInfo {
 export interface ISetDefaultSourceStylesParams extends IDataSourceParams {
   styles?: IDataSourceDefaultStyles;
   captions?: ICaptions;
+  propertiesOrder?: IPropertiesOrder;
```

Comment:
why propertiesOrder is needed here ?

Reply:
Isn't this how you set it from the UI using the setDefaultsAPI ?

Reply:
you're right just the API name is confusing "setDefaultSourceStyles".
Now it is used for style caption and properties order

Reply:
You want to rename it to just `setDefaults` ?